source/reference/minio-mc/mc-undo.rst

```diff
@@ -59,29 +60,56 @@ Parameters
    The full path to the object or prefix where the command should run.
    The path must include the :ref:`ALIAS <minio-mc-alias>`, bucket, and prefix or object name.
 
-.. mc-cmd:: --last
+.. mc-cmd:: --action
    :optional:
 
-   Accepts an integer value specifying the number of ``PUT`` and/or ``DELETE`` changes to undo.
-   
-   If not specified, the command undoes one (``1``) operation.
+   Undo the most recent change of the specified type.
+   Accepted values are ``DELETE`` or ``PUT``.
 
-.. mc-cmd:: --recursive, r
+   By default, :mc:`mc undo` reverses both ``DELETE`` and ``PUT`` operations.
+   Use :mc-cmd:`~mc undo --action` to choose one or the other, but only for the most recent operation of the specified type.
+
+   The following command undoes the most recent ``PUT`` for the file ``today.zip`` in bucket ``data``:
+
+   .. code-block:: shell
+      :class: copyable
+
+      mc undo myminio/data/today.zip --action "PUT"
+
+   This example undoes the most recent ``DELETE`` for the prefix ``archive`` and any child objects:
```

Comment:
Replacing `undoes` here, too, if you want.
Also, makes this one phrased similarly to the one above with an expansion of the recursive part of the command explanation.

```suggestion
   The following command reverses the most recent ``DELETE`` operation for the prefix ``archive`` and, recursively,  any child objects of the prefix:
```